client/webserver/site/src/js/app.ts

```diff
@@ -566,15 +572,16 @@ export default class Application {
       case 'walletstate':
       case 'walletconfig': {
         // assets can be null if failed to connect to dex server.
-        if (!this.assets) return
+        if (!this.assets) break
```

Comment:
I'm not sure this was a bug to fix.  I expect that the walletconfig note is unhandleable if `this.assets` is null.

As an aside, I don't know that the comment or this condition are even possible.  `User.Assets` can only be undefined if fetchUser has never been called, and I don't see that.  @buck54321 do you remember this?  https://github.com/decred/dcrdex/pull/989#discussion_r602382449

Reply:
It's not clear to me what's gained with this change either. Is there a specific issue you were addressing, @ukane-philemon?

I'm sure this line was written with blood, but if you think it's worth pursuing, we'd need to audit the `walletstate`and `walletconfig` notifications to ensure that they can't arrive async after we connect but before we receive our `fetchUser` result.

Reply:
> Is there a specific issue you were addressing, @ukane-philemon?

No.

Reply:
OK no sweat.  Let's just keep this part the same and I think we can get this PR merged.